playingaway.R

```diff
@@ -8,134 +8,157 @@ library(ggplot2)
 library(ggthemes)
 library(tidyr)
 
+# Get team data ----
+team_state_csv <-
+    "team,state
```

Comment:
And perhaps `team_state` and `venue_state`. It depends how you merge. I can see `team_state==venue_state` being a question you'd want to ask, so there's merit in unambiguous names. This is all "SQL thinking" ("[relational theory](http://shop.oreilly.com/product/0636920046158.do)").